detail.html

```diff
@@ -0,0 +1,40 @@
+<%inherit file="app_base.htm" />
+
+<%block name="detail">
+
+    <div class="col-8 text-center">
+        <div id="description">
+
+            <ul id="smallPics">
+            % for img in product.image_urls():
+                <% 
+                    # get the name of the pictures
+                    start  = img.rfind('/')+1
+                    finish = img.rfind('.')
+                    alt = img[start:finish]
+                %>
+                <li class="smallPics">
+                    <img src="${ img | h }" alt="${ product.name + ' - ' + alt + ' image' | h }">
+                </li>

```

Comment:
a comment in the code would be helpful here.

Reply:
I"ll go change that